examples/OAuthProducer/Program.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright 2022 Confluent Inc.

```

Comment:
Do we really need this example, it looks identical to the consumer one?

Reply:
I was following the go examples, but I guess I can merge both of them. Let me update this.

Reply:
Fair enough, I think it is better to have duplciate examples in that case, rather than having one example be both producer and consumer.